app/src/org/commcare/fragments/connect/ConnectDeliveryVisitsFragment.kt

```
@@ -0,0 +1,16 @@
+package org.commcare.fragments.connect
+
+import androidx.fragment.app.Fragment
+import org.commcare.dalvik.R
+import org.commcare.fragments.RefreshableTab
+
+class ConnectDeliveryVisitsFragment :
+    Fragment(R.layout.fragment_connect_delivery_visits),
+    RefreshableTab {
+    override fun updateView() {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)